modules/ROOT/pages/comparison-of-7-x-and-6-x.adoc

```diff
@@ -127,14 +127,15 @@ See the xref:bonita-bpm-installation-overview.adoc[Installation overview].
 | No change. See xref:mobile-portal.adoc[Mobile overview]
 
 | Subtasks
-| A subtask is a part of an existing task. The assignee can create a subtask (to request a missing information for example) and must assign it to a specific person, by name. The assignee can be the creator
-| Supported up to 7.2.4. No more substasks can be created starting from 7.3.0, but open process instances execute them correctly
+| A subtask is a part of an existing task. The assignee can create a subtask (to request missing information for example) and must assign it to a specific person, by name. The assignee can be the creator
+| Supported up to 7.2.4. No more subtasks can be created starting from 7.3.0, but open process instances execute them correctly
 
 | Replay tasks and connectors in error
 | It is now possible for the administrator to replay a task or a connector that is in error. This enables a resolution of failed tasks and better service to end users. and connectors in error
 | No change. See xref:process-configuration-overview.adoc[Process configuration overview] and xref:mobile-portal.adoc[Mobile overview]
 
 | Anonymous user
+// TODO find the xref for "anonymous user"
 | You can now complete a task as an http://documentation.bonitasoft.com/anonymous-user[anonymous user], that is, without being registered in the organization. For example, on an e-commerce site, a new user can browse stock and save items to a basket, then register with the site if they want to save their basket for later or to buy something
```

Comment:
@abirembaut do you know if we have a page to redirect to?

Reply:
it no longer exists but I added the link to the guest user feature in the 7.x column that is the equivalent in 7.x